crates/apub/src/objects/private_message.rs

```diff
@@ -130,9 +136,18 @@ impl Object for ApubPrivateMessage {
     let recipient = note.to[0].dereference(context).await?;
     PersonBlock::read(&mut context.pool(), recipient.id, creator.id).await?;
 
+    // If its a local user, check that they can receive private messages
+    if recipient.local {
```

Comment:
This check is not really needed because the recipient of a federated private message is always a local user.

Reply:
K fixed.